docs/Changelog.md

```diff
@@ -29009,6 +29009,77 @@ This version of the operator has been available since version 23 of the default
 <dd>The type of the input `y_zero_point` and the output `y`.</dd>
 </dl>
 
+### <a name="RMSNormalization-23"></a>**RMSNormalization-23**</a>
+
+  This is RMS normalization defined in ONNX as function as described in the paper https://arxiv.org/pdf/1910.07467.
+        The overall computation can be split into two stages. The root mean squared norm is taken over the last D dimensions,
+        where D is the dimension of normalized_shape. For example, if normalized_shape is (3, 5) (a 2-dimensional shape),
+        the rms norm is computed over the last 2 dimensions of the input. The computation required by standardization can be
+        described by the following equations.
+        ```
+        XSquared = Mul(X, X)
+        XSquaredMean = ReduceMean<axes=normalized_axes>(XSquared)
+        RMSEps = Add(XSquaredMean, epsilon)
```

Comment:
Nit: Suggest renaming `RMSEps` to something else, maybe `MeanSquareEpsilon`. The `R` in `RMS` stands for `(square) root`, so it seems inappropriate to use it before the square-root is taken.